src/classes/test/DatabaseManager.js

```diff
@@ -52,47 +53,52 @@ test.serial('add status', async (t) => {
     })
   }
   t.is(await t.context.DatabaseManager.addStatus('add-stats-name', 1, true), 0)
+  t.true(t.context.qbRun.calledOnce)
 })
 
 test.serial('get default settings', async (t) => {
-  await t.context.DatabaseManager.getDefaultStatus()
+  const value = await t.context.DatabaseManager.getDefaultStatus()
+  t.deepEqual(value, { name: 'add-stats-name', type: 1 })
```

Comment:
add-statuses-name?
also why is this called get default settings, it just gets the default status